### PR TITLE
refactor(orc8r): correct swagger spec for `networksNetworkIdPrometheusAlertReceiverRouteGet`

### LINKED ISSUE #13474: Investigate (and correct) Swagger type for `networksNetworkIdPrometheusAlertReceiverRouteGet`
- `/nms/app/state/AlarmsApiUtil.ts`
  - `MagmaAPI.alerts.networksNetworkIdPrometheusAlertReceiverRouteGet`

### PR DESCRIPTION
**Pull request by Neudrino**
## Summary

- Fixes https://github.com/magma/magma/issues/13474

## Test Plan

- `.build.py --generate`
- `yarn tsc`
- CI

## Additional information

- the `cloud-workflow` seems to be flaky :crossed_fingers: 
- the `cloud-workflow` has a strange interaction with the totally unrelated `Semantic PR` check

### CODE REVIEW

**Review by thmsschmitt — APPROVED** [review]
NMS changes look good

File: orc8r/cloud/go/services/metricsd/obsidian/models/swagger.v1.yml [hunk 1]
```diff
@@ -1149,18 +1149,12 @@ definitions:
...
-            type: string
+        additionalProperties:
+          type: string
       match_re:
         type: object
-        properties:
-          label:
-            type: string
-          value:
-            type: string
```
Comment (sebathomas) [inline orc8r/cloud/go/services/metricsd/obsidian/models/swagger.v1.yml:1163]: It looks like both of them do have the properties `label` and `value` on the Go side, and this is not touched in this PR. Why did you change the interface?

Comment (sebathomas) [reply orc8r/cloud/go/services/metricsd/obsidian/models/swagger.v1.yml:1163]: As discussed offline, it seems the generated types in the backend are not used, and the new interface is correct.

**Review by sebathomas — APPROVED** [review]
